### PR TITLE
[AllBundle] Fix unused method, property, variable or parameter

### PR DESCRIPTION
**Pull request by cv65kr**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | yes
| Deprecations? | yes
| Fixed tickets | no

Fix unused method, property, variable or parameter from Insight Sensiolabs.

### CODE REVIEW

File: src/Kunstmaan/AdminBundle/Toolbar/BundleVersionDataCollector.php [hunk 1]
```diff
@@ -11,19 +11,25 @@
 
 class BundleVersionDataCollector extends AbstractDataCollector
 {
-    /** @var VersionChecker */
+    /**
+     * @var VersionChecker
+     */
     private $versionChecker;
```
Comment (sandergo90) [inline src/Kunstmaan/AdminBundle/Toolbar/BundleVersionDataCollector.php]: This does not need to change.

File: src/Kunstmaan/AdminBundle/Toolbar/BundleVersionDataCollector.php [hunk 2]
```diff
@@ -11,19 +11,25 @@
...
 
-    /** @var Logger */
-    private $logger;
-
-    /** @var Cache */
+    /**
+     * @var Cache
+     */
```
Comment (sandergo90) [inline src/Kunstmaan/AdminBundle/Toolbar/BundleVersionDataCollector.php]: Idem

**Comment by cv65kr** [comment]
@sandergo90, done.

**Comment by sandergo90** [comment]
Thanks! Can you also rebase to 6.0 branch ? It's BC breaking so needs to be on the next major release.

**Comment by cv65kr** [comment]
@sandergo90, sure ready :)

**Comment by sandergo90** [comment]
And now a squash and we are ready ! ;)

**Pull request merged**